platform-core-module/src/main/java/com/softicar/platform/core/module/standard/configuration/ProgramStandardConfiguration.java

```diff
@@ -1,50 +1,72 @@
 package com.softicar.platform.core.module.standard.configuration;
 
-import com.softicar.platform.common.code.reference.point.SourceCodeReferencePoints;
 import com.softicar.platform.core.module.configuration.AbstractStandardConfiguration;
 import com.softicar.platform.core.module.program.AGProgram;
-import com.softicar.platform.core.module.program.IProgram;
 import com.softicar.platform.core.module.program.Programs;
 import com.softicar.platform.core.module.program.execution.scheduled.AGScheduledProgramExecution;
+import com.softicar.platform.core.module.program.state.AGProgramState;
 import com.softicar.platform.core.module.uuid.AGUuid;
 import com.softicar.platform.core.module.uuid.AGUuidBasedSourceCodeReferencePoint;
-import com.softicar.platform.db.core.transaction.DbTransactions;
+import java.util.Collection;
+import java.util.Optional;
+import java.util.stream.Collectors;
 
 public class ProgramStandardConfiguration extends AbstractStandardConfiguration {
 
 	@Override
 	public void createAndSaveAll() {
 
-		Programs//
+		var programs = createPrograms();
+
+		saveAll(programs);
+		saveAll(createStates(programs));
+		saveAll(createScheduledExecutions(programs));
+	}
```

Comment:
Out of curiosity: How much time did we save here?

Reply:
I did not measure this in isolation, I rather looked at the amount of statements, and this were a lot of statements, including unnecessary BEGIN, COMMIT and SELECT statements.